var/spack/repos/builtin/packages/hpx/package.py

```diff
@@ -21,7 +21,7 @@ class Hpx(CMakePackage, CudaPackage, ROCmPackage):
     tags = ["e4s"]
 
     version("master", branch="master")
-    version("stable", tag="stable", commit="38d5bf935e5a49f9466c5e615e04e8d553a73dc6")
+    version("stable", tag="stable", commit="103a7b8e3719a0db948d1abde29de0ff91e070be")
```

Comment:
Is this useful to keep? Might as well remove it if the stable tag isn't ... stable :smirk: